python_modules/libraries/dagster-dg/dagster_dg/cli/list.py

```diff
@@ -175,21 +154,35 @@ def list_plugins_command(
 )
 @dg_global_options
 @cli_telemetry_wrapper
-def list_component_type_command(output_json: bool, **global_options: object) -> None:
-    """List registered Dagster components in the current project environment."""
+def list_plugins_command(
+    name_only: bool,
+    plugin: Optional[str],
+    feature: Optional[PluginObjectFeature],
+    output_json: bool,
+    **global_options: object,
+) -> None:
+    """List dg plugins and their corresponding objects in the current Python environment."""
     cli_config = normalize_cli_config(global_options, click.get_current_context())
     dg_context = DgContext.for_defined_registry_environment(Path.cwd(), cli_config)
     registry = RemotePluginRegistry.from_dg_context(dg_context)
 
     if output_json:
         output: list[dict[str, object]] = []
-        for entry in sorted(
-            registry.get_objects(feature="component"), key=lambda x: x.key.to_typename()
-        ):
-            output.append({"key": entry.key.to_typename(), "summary": entry.summary})
+        for entry in sorted(registry.get_objects(), key=lambda x: x.key.to_typename()):
+            output.append(
```

Comment:
totally fine for the purposes of this PR but my dream is for the "DagsterOuterTable / DagsterInnerTable" stuff to have functionality to auto-generate the json-ified output based on the table structure

e.g. right now this output is still kinda bad because it doesn't include the plugin name

this PR doesn't make it worse or anything, just logging for the future